### PR TITLE
fix trigger time. fixes #61, set default pre trigger time to 55ns (AR…

### LINKED ISSUE #61: Trigger times not set correctly
It looks like the trigger simulation modules set the trigger time to the start of the coincidence window around the pulse instead of the pulse.

### PR DESCRIPTION
**Pull request by cg-laser**
…IANNA default)

### CODE REVIEW

**Comment by christophwelling** [comment]
Don't the other trigger modules need the same fix?

**Comment by cg-laser** [comment]
the simple trigger module already used the function from the high/low module. Fixed the multiHighLowTrigger module too.

File: NuRadioReco/modules/trigger/multiHighLowThreshold.py [hunk 1]
```diff
@@ -1,21 +1,21 @@
 from NuRadioReco.utilities import units
 from NuRadioReco.framework.parameters import stationParameters as stnp
-from NuRadioReco.framework.trigger import HighLowTrigger
```
Comment (christophwelling) [inline NuRadioReco/modules/trigger/multiHighLowThreshold.py]: I'm pretty sure this import is needed. Later a HighLowTrigger object is created.

Comment (cg-laser) [reply NuRadioReco/modules/trigger/multiHighLowThreshold.py]: thanks for spotting this! I somehow thought it was an outdated import of an old trigger module... I put it back in

**Comment by cg-laser** [comment]
@anelles given that the bug I introduced was not caught by the built bot, can we add all trigger module to the example?

**Pull request merged**